src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -234,6 +241,37 @@ const Actions: React.FC<Props> = ({
         };
     };
 
+    const mapsLinkForParcels = (): string => {
+        let mapsLinkString = "https://www.google.com/maps/dir";
+        selectedParcels.forEach((parcel) => {
+            mapsLinkString = mapsLinkString.concat("/", parcel.addressPostcode);
+        });
+        mapsLinkString = mapsLinkString.concat("//");
+        return mapsLinkString;
```

Comment:
Instead of concatnating strings, we can simply that using `map` and `join`.
```suggestion
        return "https://www.google.com/maps/dir/" + selectedParcels.map(parcel => parcel.addressPostcode).join("/") + "//"
```

- Also we should deal with the cases where `parcel.addressPostcode` has a whitespace(s)
- Would this function be better named as `mapLinkForSelectedParcels`? `mapLinkForParcels` kind of implies we need to pass a list of parcels as a parameter, or we have a defined set of parcels that doesn't change.